.github/workflows/ci_eval.yaml

```diff
@@ -53,18 +53,13 @@ jobs:
           python -m pip install --no-compile --upgrade pip
 
           # Note: We install in three steps in order to satisfy requirements
-          # from non default locations first. Installing the PyTorch CPU
-          # wheels saves multiple minutes and a lot of bandwidth on runner setup.
+          # from non default locations first.
           pip install --no-compile -r pytorch-cpu-requirements.txt
+          pip install -r requirements-iree-unpinned.txt
+          pip install --no-compile \
+            -r sharktank/requirements-tests.txt \
```

Comment:
As above.

Reply:
I'm not following here. What change do you want made?

I believe
```
          pip install --no-compile \
            -r sharktank/requirements-tests.txt \
            -e sharktank/
```

is sufficient, no need to also install `sharktank/requirements.txt` or `sharktank/requirements-dev.txt`

Reply:
As commented above with this, this drops installing the dependencies specified in `sharktank/requirements-dev.txt`. Just wanted to point out that this is the same here but that might have been a little to short. No change needed.